_source/logzio_collections/_metrics-sources/postgres-metrics.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Ship Azure PostgreSQL metrics
+logo:
+  logofile: postgresql.svg
+  orientation: vertical
+data-source: Azure PostgreSQLs
+contributors:
+  - yotamloe
+  - shalper
+shipping-tags:
+  - azure
+
+---
+
+To monitor your Azure service metrics,
+we recommend configuring your services
+to send their metrics to Azure Monitor.
+When you set up Metricbeat using the configuration on this page,
+Metricbeat will collect metrics from Azure PostgreSQLs
```

Comment:
Metricbeat will collect metrics from Azure PostgreSQLs
With an "s" at the end. Is that was you meant?